starknet/src/types/proposal.cairo

```diff
@@ -16,15 +16,26 @@ const TWO_POWER_32: u128 = 0x100000000;
 const TWO_POWER_64: u128 = 0x10000000000000000;
 const TWO_POWER_96: u128 = 0x1000000000000000000000000;
 
+/// Definition of a proposal.
```

Comment:
Data stored when a proposal is created. The proposal state at any time consists of this struct along with the corresponding for, against, and abstain vote counts. The proposal status is a function of the proposal state as defined in the execution strategy for the proposal.

Reply:
e436b9d57c73975e3bece13603cc9ef4d5deeae0